### DISCUSSION TITLE
Can set "Preview Markdown" as default when open a markdown snippet ?

### DISCUSSION THREAD

**Discussion by i-bugmaker** [massCodeIO/massCode#46] [Ideas]
Everytime I open a markdown snippet, it is a "source code" mode.

Can set "Preview Markdown" as default when open a markdown snippet ? I think that's good for reading.

version:2.2.0

**Comment by antonreshetov** [comment]
@i-bugmaker I'll think about it. But I guess the "opposite side of the camp" will be annoyed as you are in this case, because it's primarily a snippet manager, ie, the source code is preferable.

**Reply by i-bugmaker** [reply]
I think it would be nice to add a Settings Option based on each person's habits so that you get the best of both worlds

**Reply by antonreshetov** [reply]
I thought of that after I pressed Enter :)

**Reply by Faq** [reply]
This would be neat feature in preferences. To implement it, we should make and issue with feature tag?
